hw/opentitan/ot_usbdev.c

```diff
@@ -67,7 +67,10 @@
 #define OT_USBDEV_BUFFER_COUNT        32u
 
 /* Time in milliseconds for a bus reset to complete. */
-#define USBDEV_BUS_RESET_TIME_MS 10
+#define OT_USBDEV_BUS_RESET_TIME_MS 10u
+
+/* Time in nanosecond of a frame */
```

Comment:
```suggestion
/* Time in nanoseconds of a frame */
```
(also: is this defined anywhere? Is this OT-specific / spec-specific / arbitrary?)

Reply:
It's standard. I added a comment